src/Router.jsx

```diff
@@ -22,6 +23,7 @@ const Router = () => {
           <Route path="/login" element={<Login />} />
           <Route path="/mypage" element={<MyPage />} />
           <Route path="/payment" element={<Payment />} />
+          <Route path="/payresult" element={<PayResult />} />
```

Comment:
일반적으로 routing 경로에서 여러 개의 단어를 사용하는 방식은 kebab-case입니다.
```suggestion
          <Route path="/pay-result" element={<PayResult />} />
```

Reply:
수정했습니다!